core/lib/resources/presence/presence_manager.js

```diff
@@ -51,16 +51,18 @@ PresenceManager.prototype.setup = function() {
   this.sentryListener = function (sentry) {   
     
     var socketIds = store.socketsForSentry(sentry);
+    logging.info('#presence - #sentry down with ' + socketIds.length +
+      ' clients ', socketIds);
+    if (manager.destroying) {
+      logging.info('#presence - manager.destroying true');
+    }
+    if (!socketIds.length || manager.destroying) {
+      return;
+    }
 
     var chain = function (socketIds) {
       var socketId = socketIds.pop();
 
-      if (!socketId || manager.destroying) {
```

Comment:
we still need the early exit in case manager is getting destroyed.

This is the case:
We initiate this chain of client offlines for a resource A.
While this is happening resource A is destroyed because the last subscriber goes away or unsubscribes.
In that case, the next chain will see that manager is already destroyed.